content/manuals/security/for-admins/hardened-desktop/settings-management/compliance-reporting.md

```diff
@@ -0,0 +1,65 @@
+---
+description: Understand how to use the Desktop settings reporting dashboard
+keywords: Settings Management, docker desktop, hardened desktop, reporting, compliance
+title: Desktop settings reporting
+linkTitle: Desktop Settings Reporting
+weight: 30
+---
+
+Desktop settings reporting is a feature of Desktop Settings Management that tracks and reports user compliance with the settings that are assigned to them. This lets administrators to track the application of settings and monitor what actions they need to take to make users compliant.
+
+This guide provides steps for accessing the reporting dashboard, viewing compliance status, and resolving non-compliant users.
```

Comment:
Is "dashboard" the right word?

Reply:
@aevesdocker @ajthilakan curious for your thoughts